translate/storage/pypo.py

```diff
@@ -48,6 +48,29 @@
 po_escape_map = dict([(value, key) for (key, value) in po_unescape_map.items()])
 
 
+def splitlines(text):
+    """Split lines based on first newline char.
+
+    Can not use univerzal newlines as they match any newline like
+    character inside text and that breaks on files with unix newlines
+    and LF chars inside comments.
+    """
+    # Find first newline
+    newline = b'\n'
+    for i, ch in enumerate(text):
+        if ch in ('\n', 10):
+            newline = b'\n'
```

Comment:
This line is unnecessary because this value is already set before the `for` loop.